Modules/Bridge/NumPy/include/itkPyBuffer.hxx

```diff
@@ -136,7 +125,7 @@ PyBuffer<TImage>::_GetImageViewFromArray(PyObject * arr, PyObject * shape, PyObj
   importer->SetImportPointer(data, numberOfPixels, importImageFilterWillOwnTheBuffer);
 
   OutputImagePointer output = TImage::New();
-  output->SetRegions(region);
+  output->SetRegions(size);
```

Comment:
You seem to be mixing up `region` and `size`.

Reply:
Me? `itk::Image` _does_ support specifying its regions by their `itk::Size`! 😃 
https://github.com/InsightSoftwareConsortium/ITK/blob/a434e3f71829f23cd748a1694bc64455aecceabd/Modules/Core/Common/include/itkImageBase.h#L340 

If the index position of a region is just (0, 0), only its size matters.